### DISCUSSION TITLE
G-Helper keeps recognizing HDR-Signal even though its off

### DISCUSSION THREAD

**Discussion by Trnt1337** [seerge/g-helper#3150] [General]
Hey,

suddenly in the main menu of g-helper i only get to see the visual modes by clicking on the "display modes are not available when hdr is enabled" button, in the past that butten only was there when i had hdr oenabled in the windows 11 settings, now is see it permanently even though i have hdr off - any solution to this?

![image](https://github.com/user-attachments/assets/d660eba6-e635-43bd-9fdf-36faf1e684ca)
![image](https://github.com/user-attachments/assets/62acf172-6c59-4216-96e6-690aa06ca822)

Thanks!!!!

**Comment by seerge** [comment]
@Trnt1337 hello, do you have `System -> Display -> Advanced Display -> Automatically Manage Color for Apps` setting enabled in windows by any chance ? 

If no - please upload the log from the app or fill in [a new issue properly according to template](https://github.com/seerge/g-helper/issues). 

Thanks

**Reply by Trnt1337** [reply]
Thanks for the fast reply! i actually had it enabled, disabling it solved the issue. I had it enabled because without it i get color banding in windows, i guess im supposed to keep that setting off though?

**Reply by seerge** [reply]
No idea, on my laptop it makes all colors look dull. Also it makes windows falsely report display as 10bit color (i.e. HDR) and confuse G-Helper
